src/pretalx/static/agenda/scss/_speaker.scss

```diff
@@ -27,6 +27,18 @@
   flex-direction: row;
 
   .talk-content {
+    .talk-slots {
+      padding: 0;
+      list-style: none;
+
+      .talk-slot {
+        color: lighten($gray-800, 25%);
```

Comment:
Hmm, I think making a shared class for talk slots would be better than repeating this color styling here.